tests/sensitivities/test_use_cases.py

```diff
@@ -0,0 +1,90 @@
+"""Example use cases for fmudesign"""
+
+import pandas as pd
+from fmu.tools.sensitivities import excel2dict_design, DesignMatrix
+
+
+def test_prediction_rejection_sampled_ensemble(tmpdir):
+    """Test making a design matrix for prediction realizations based on an
+    ensemble made with manual history matching (rejection sampling).
+
+    In the use-case this test is modelled on, the design matrix is used
+    to set up a prediction ensemble where each DATA file points to another
+    Eclipse run on disk which contains the history, identified by the
+    realization index ("HMREAL") in the history match run.
+    """
+    tmpdir.chdir()
+    general_input = pd.DataFrame(
+        data=[
+            ["designtype", "onebyone"],
+            ["repeats", 3],  # This matches the number of HM-samples we have.
+            ["rms_seeds", None],  # Geogrid from HM realization is used
+            ["background", "hmrealizations.xlsx"],
+            ["distribution_seed", None],
+        ]
+    )
+    defaultvalues = pd.DataFrame(
+        columns=["param_name", "default_value"],
+        data=[
+            # Currently all background parameters must be mentioned in
+            # DefaultValues, this requirement might be lifted later.
+            ["HMREAL", "-1"],
+            ["ORAT", 6000],
+            ["RESTARTPATH", "FOO"],
+            ["HMITER", "-1"],
+        ],
+    )
+
+    # Background to separate file, these define some history realizations that
+    # all scenarios should run over:
+    pd.DataFrame(
+        columns=["RESTARTPATH", "HMREAL", "HMITER"],
+        data=[
+            ["/scratch/foo/2020a_hm3/", 31, 3],
+            ["/scratch/foo/2020a_hm3/", 38, 3],
+            ["/scratch/foo/2020a_hm3/", 54, 3],
+        ],
+    ).to_excel("hmrealizations.xlsx")
+
+    writer = pd.ExcelWriter("designinput.xlsx", engine="openpyxl")
+    general_input.to_excel(writer, sheet_name="general_input", index=False, header=None)
+    pd.DataFrame(
+        columns=[
+            "sensname",
+            "numreal",
+            "type",
+            "param_name",
+            "dist_name",
+            "dist_param1",
+            "dist_param2",
+        ],
+        data=[
+            ["ref", None, "ref", None],
```

Comment:
I wonder if the sensitivity type here should be "background" instead of "ref". (The sensname can still be "ref" of course). The difference is that the intention with a "ref" sensitivity is that it is normally only one realisation, and would not be plotted in the tornado plot, but only used as a reference to calculate the deltas in the plot. The background sensitivity will pop up as a sensitivity in the plot and you can see the P10/P90 variation in the background realisations. This might be useful to compare the variation of the scenario sensitivities to the variation in the background (ref) realisations. But I leave this to you and your use case here. The test looks fine.

Reply:
changing the second `"ref"` to `"background"`